### DISCUSSION TITLE
The meaning of the prefix of filename xxx.avplayer.js

### DISCUSSION THREAD

**Discussion by Kuo-TingKai** [zhaohappy/libmedia#23] [Q&A]
Dear author.
I would like to know the meaning and the difference between the prefixed of the filenames {xxx}.avplayer.js, thank you.

**Comment by zhaohappy** [comment]
These files with prefix are the compiled output of webpack. And the prefix is ​​generated by the algorithm inside webpack. It means it is a dynamically imported module when we use ```const a = import('xxxx')``` . Here we turn different demux、mux、ioloader modules (these modules are not used at the same time) into dynamic modules to reduce the size of the main file(avplayer.js). They need to be in the same directory with the main file.
